src/aics-image-viewer/components/GlobalVolumeControls.tsx

```diff
@@ -175,14 +193,15 @@ const STYLES = {
   slidersWrapper: {
     width: "calc(100% - 20px)",
     margin: "auto",
+    paddingTop: "18px",
   },
   controlRow: {
     height: "3em",
     display: "flex",
   },
   controlName: {
     flex: 2,
-    whiteSpace: "nowrap",
+    "white-space": "nowrap",
```

Comment:
really?

Reply:
Both are correct, but the first didn't typecheck. I looked into this, and it seems `whiteSpace` has a union type of string literals for all the allowable values of this property, but the type checker has no idea about that when it decides on a type for `STYLES`, so that property just gets the `string` type. There might be other solutions for this (putting this style inline, putting it in an actual stylesheet, maybe making `STYLES` read only?) but this was the quickest.

Reply:
it's a minor thing but consider adding a little comment saying that the typechecker complains about the camelCased version of this

Reply:
Ah, it's also possible to say `whiteSpace: "nowrap" as "nowrap"`. Any opinions as to which is cleaner?

Reply:
very slight preference for the latter